applications/GeoMechanicsApplication/custom_workflows/adaptive_time_incrementor.hpp

```diff
@@ -0,0 +1,50 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//                   Anne van de Graaf
+//
+
+#pragma once
```

Comment:
Just for my information: When do we use .hpp and when do we use .h in this project?

Reply:
To the best of my knowledge, there is no clear policy about the file extension to be used for header files. Let's first double-check the Kratos Style guide, just to make sure that we didn't overlook it. If it is not stated there, we may want to define a policy in our supplementary coding conventions.

Reply:
I don't see anything in the Kratos Style guide, so let's indeed discuss later